modules/dynamic-plugins/proc-topology-configure.adoc

```diff
@@ -0,0 +1,326 @@
+= Configuration
+
+== Viewing OpenShift routes
+To view OpenShift routes, you must grant read access to the routes resource in the Cluster Role:
+
+[source,yaml]
+----
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: backstage-read-only
+  rules:
+    ...
+    - apiGroups:
+        - route.openshift.io
+      resources:
+        - routes
+      verbs:
+        - get
+        - list
+----
+
+You must also add the following in kubernetes.customResources property in your `app-config.yaml` file:
```

Comment:
`kubernetes.customResources`

Check other files in this PR for `kubernetes.customResources` Sometimes "kubernetes is also misspelled, like missing an "e"